frontend/src/routes/Router.jsx

```diff
@@ -49,7 +49,7 @@ try {
   ChatAppLayout =
     require("../plugins/app-deployment/chat-app/ChatAppLayout.jsx").ChatAppLayout;
 } catch (err) {
-  TrialRoutes = NotFound;
+  AppDeployments = NotFound;
```

Comment:
@athul-rs Why are we assigning the `NotFound` component to `AppDeployments` alone on failure? If the component is not available in the expected location, it will fail gracefully, meaning the route itself won't get added. If this happens, it will automatically render the `NotFound` page based on our current implementation.

Reply:
@tahierhussain Refactored to support multiple plugins coming in future.